src/components/site-header/SiteHeaderNavMenu.jsx

```diff
@@ -25,15 +21,13 @@ const SiteHeaderNavMenu = () => {
           description: 'Dashboard link title in site header navigation.',
         })}
       </NavLink>
-      {!contentHighlightsConfiguration.canOnlyViewHighlightSets && (
-        <NavLink to={`/${activeEnterpriseCustomer.slug}/search`} className={mainMenuLinkClassName}>
-          {intl.formatMessage({
-            id: 'site.header.nav.search.title',
-            defaultMessage: 'Find a Course',
-            description: 'Find a course link in site header navigation.',
-          })}
-        </NavLink>
-      )}
+      <NavLink to={`/${activeEnterpriseCustomer.slug}/search`} className={mainMenuLinkClassName}>
```

Comment:
Removing the conditional "Find a Course" given the conditional logic was incorrect before. It should be based on whether learner is an assignments only learner. We will come back to this as a standalone ticket.